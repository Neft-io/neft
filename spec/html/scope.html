<unit name="elema">
	<unit name="elemb"><b></b></unit>
	<elemb></elemb>
</unit>

<elema></elema>
<elemb></elemb>