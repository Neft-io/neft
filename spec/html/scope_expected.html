<unit><unit><b></b></unit></unit><elemb></elemb>